spec/project/default_config_spec.rb

```diff
@@ -47,7 +47,7 @@ def cop_configuration(config_key)
       .to match_array([*config_keys, 'Metrics/BlockLength'])
   end
 
-  it 'sorts configuration keys alphabetically', :pending do
+  it 'sorts configuration keys alphabetically' do
```

Comment:
```suggestion
  it 'sorts configuration keys alphabetically', pending: 'sort putting sub-departments below' do
```

Reply:
I tried rewriting this spec in #1507. That PR could replace this commit.